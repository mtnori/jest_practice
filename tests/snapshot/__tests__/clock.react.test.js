import React from 'react';
import renderer from 'react-test-renderer';
import Clock from '../Clock.react';

jest.useFakeTimers();
Date.now = jest.fn(() => 1482363367071);

it('正しいレンダリング', () => {
  const tree = renderer.create(<Clock />).toJSON();
  expect(tree).toMatchSnapshot();
});
