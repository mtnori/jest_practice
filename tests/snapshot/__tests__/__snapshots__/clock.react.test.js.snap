// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`正しいレンダリング 1`] = `
<p>
  1482363367.071
   seconds have ellapsed since the UNIX epoch.
</p>
`;
